### DISCUSSION TITLE
I can't add my ssh key

### DISCUSSION THREAD

**Discussion by Leooo2233** [community/community#57770] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I get the following error:

$ ssh -T git@github.com
git@github.com: Permission denied (publickey).

**Comment by airtower-luna** [comment]
That command is for testing if your key works, for *adding* it in the first place see here: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

If you have already done that use `ssh -vT git@github.com` (note the extra `v` for "verbose") to get debug info. And if you need help figuring out what it means, post the log here. :slightly_smiling_face:
